eradiate/validators.py

```diff
@@ -47,26 +49,39 @@ def path_exists(_, attribute, value):
     an existing target. Raises a ``FileNotFoundError`` otherwise.
     """
     if not value.exists():
-        raise FileNotFoundError(f"{attribute} points to '{str(value)}' "
-                                f"(path does not exist)")
+        raise FileNotFoundError(
+            f"{attribute} points to '{str(value)}' " f"(path does not exist)"
+        )
 
 
 def is_file(_, attribute, value):
     """Validates if ``value`` is a :class:`pathlib.Path` and points to
     an existing file. Raises a ``FileNotFoundError`` otherwise.
     """
     if not value.is_file():
-        raise FileNotFoundError(f"{attribute} points to '{str(value)}' "
-                                f"(not a file)")
+        raise FileNotFoundError(
+            f"{attribute} points to '{str(value)}' " f"(not a file)"
+        )
+
+
+def is_path(_, attribute, value):
+    """Validates if ``value`` is a :class:`pathlib.Path` and points to
+    an existing file. Raises a ``FileNotFoundError`` otherwise.
+    """
+    if value is None or not os.path.exists(value):
+        raise FileNotFoundError(
+            f"{attribute} points to '{str(value)}' " f"(not a file)"
+        )
```

Comment:
Please use the pathlib interface here:
```suggestion
def is_path(_, attribute, value):
    """Validates if ``value`` is a :class:`pathlib.Path` and points to
    an existing file. Raises a ``FileNotFoundError`` otherwise.
    """
    if not value.exists():
        raise FileNotFoundError(
            f"{attribute} points to '{str(value)}' (does not exist)"
        )
```